dev/how-to/troubleshoot/tidb-lightning.md

```diff
@@ -67,7 +71,7 @@ tidb-lightning-ctl --config conf/tidb-lightning.toml --checkpoint-error-destroy=
 
 See the [Checkpoints control](/dev/reference/tools/tidb-lightning/checkpoints.md#checkpoints-control) section for other options.
 
-## ResourceTemporarilyUnavailable("Too many open engines …: 8")
+## ResourceTemporarilyUnavailable("Too many open engines …: …")
 
 **Cause**: The number of concurrent engine files exceeds the limit imposed by `tikv-importer`. This could be caused by misconfiguration. Additionally, if `tidb-lightning` exited abnormally, an engine file might be left at a dangling open state, which could cause this error as well.
```

Comment:
```suggestion
**Cause**: The number of concurrent engine files exceeds the limit specified by `tikv-importer`. This could be caused by misconfiguration. Additionally, if `tidb-lightning` exited abnormally, an engine file might be left at a dangling open state, which could cause this error as well.
```